Applications/spire/include/spire/time_and_sales/time_and_sales_window.hpp

```diff
@@ -82,6 +82,7 @@ namespace spire {
       bool m_h_scrolling;
       QTimer* m_v_scroll_bar_timer;
       QTimer* m_h_scroll_bar_timer;
+      QFont m_header_font;
```

Comment:
I don't think having this font is helpful, it appears to be duplicating the font in the horizontal header. Instead of duplicating its value, refer to the existing font via horizontalHeader()->font();